src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Kindly remove this eslint disable line and fix the issue in the code.

Reply:
Okay I fixed it